Modules/Core/Common/include/itkTriangleCell.h

```diff
@@ -188,9 +188,9 @@ class ITK_TEMPLATE_EXPORT TriangleCell
   // This was observed in at least gcc 4.8 and 5.4.0, and
   // AppleClang 7.0.2 and 8.0.0. Probably others too.
   // "= default" doesn't gain us much, so just don't use it here.
-  ~TriangleCell() override{};
+  ~TriangleCell() override{}
 #else
-  ~TriangleCell() override = default;
+  ~TriangleCell() override = default
```

Comment:
This ; after default should be necessary, I think you maybe removed it in error. Your compiler probably isn't getting into this #else.